Assets/Tests/InputSystem.Editor/XRIPackageTest.cs

```diff
@@ -28,6 +28,12 @@ public IEnumerator TearDown()
         {
             yield return null;
         }
+
+        //Delete the Assets/XRI folder (and its content) that the XRI package creates
+        if (AssetDatabase.IsValidFolder("Assets/XRI"))
+        {
+            AssetDatabase.DeleteAsset("Assets/XRI");
```

Comment:
I'm not clear from the documentation if the successfully deletes a folder containing files or if this only works if the folder is empty
https://docs.unity3d.com/ScriptReference/AssetDatabase.DeleteAsset.html

Assuming this is tested and does what is expected then I'm happy to approve this quite simple change